### DISCUSSION TITLE
AutoType Code [Feature Suggestion]

### DISCUSSION THREAD

**Discussion by yabberish** [community/community#25840] [Actions]
<p>Hello!</p>
<p>Recently, I’ve been thinking what if GitHub auto-types stuff for you?</p>
<p>For example, you hit the button <code>(</code> and you instantly get <code>()</code></p>
<p>The same thing happens with ", etc. </p>
<p>A lot of very popular programs have this feature, such as visual studio code!</p>
<p>This will be a very helpful feature because it will save time for a lot of people!</p>
<p>Thanks!</p>
<p>Elflanded.</p>

**Comment by ethomson** [comment, marked as answer]
<p>Thanks for the suggestion, we’ll keep it in mind!</p>

**Comment by 5factor** [comment]
<p>Having basic intellisense would be a nice feature but I don’t think it is something that GitHub should prioritze. GitHub isn’t a code editor and editing code on the website instead of a code editor isn’t great practice anyways. The best way to edit code is to use Git or the GitHub Desktop application to sync code between your computer and the website.</p>
